users/config.rst

```diff
@@ -298,100 +305,145 @@ One or more ``folder`` elements must be present in the file. Each element
-    and unshare the folder (subject to skipIntroductionRemovals being false on the introducer device).
+.. option:: folder.device
+
+    These must have the ``id`` attribute and can have an ``introducedBy``
+    attribute, identifying the device that introduced us to share this folder
+    with the given device.  If the original introducer unshares this folder with
+    this device, our device will follow and unshare the folder (subject to
+    :opt:`skipIntroductionRemovals` being ``false`` on the introducer device).
+
     All mentioned devices are those that will be sharing the folder in question.
     Each mentioned device must have a separate ``device`` element later in the file.
     It is customary that the local device ID is included in all folders.
     Syncthing will currently add this automatically if it is not present in
     the configuration file.
 
-minDiskFree
-    The minimum required free space that should be available on the disk this folder
-    resides. The folder will be stopped when the value drops below the threshold. Accepted units are
-    ``%``, ``kB``, ``MB``, ``GB`` and ``TB``. Set to zero to disable.
+    The ``encryptionPassword`` sub-element contains the secret needed to decrypt
+    this folder's data on the remote device.  If left empty, the data is plainly
+    accessible (but still protected by the transport encryption).  The mechanism
+    and how to set it up is described in a :doc:`separate chapter <untrusted>`.
+
+.. option:: folder.minDiskFree
+
+    The minimum required free space that should be available on the disk this
+    folder resides.  The folder will be stopped when the value drops below the
+    threshold.  The element content is interpreted according to the given
+    ``unit`` attribute.  Accepted ``unit`` values are ``%`` (percent of the disk
+    / volume size), ``kB``, ``MB``, ``GB`` and ``TB``.  Set to zero to disable.
+
+.. option:: folder.versioning
 
-versioning
     Specifies a versioning configuration.
 
-.. seealso::
-    :ref:`versioning`
+    .. seealso::
+	:doc:`versioning`
+
+.. option:: folder.copiers
+	    folder.hashers
 
-copiers, pullers, hashers
-    The number of copier, puller and hasher routines to use, or ``0`` for the
-    system determined optimums. These are low level performance options for
+    The number of copier and hasher routines to use, or ``0`` for the
+    system determined optimums. These are low-level performance options for
     advanced users only; do not change unless requested to or you've actually
     read and understood the code yourself. :)
 
-order
+.. option:: folder.pullerMaxPendingKiB
+
+    Controls when we stop sending requests to other devices once we’ve got this
+    much unserved requests.  The number of pullers is automatically adjusted
+    based on this desired amount of outstanding request data.
+
+.. option:: folder.order
+
     The order in which needed files should be pulled from the cluster.
```

Comment:
```suggestion
    The order in which needed files should be pulled from the cluster. 
    It has no effect when the folder type is "send only".
```

Reply:
Just one quick addition I'd like to suggest (actually taken from https://github.com/syncthing/docs/pull/532).